api/src/main/java/com/okta/sdk/client/ClientBuilder.java

```diff
@@ -306,9 +312,51 @@
      *
      * @param authenticationScheme the type of authentication to be used for communication with the Okta API server.
      * @return the ClientBuilder instance for method chaining
+     *
+     * @deprecated use {@link #setAuthorizationMode(AuthorizationMode)} to indicate the authentication scheme.
      */
+    @Deprecated
     ClientBuilder setAuthenticationScheme(AuthenticationScheme authenticationScheme);
 
+    /**
+     * Allows specifying an authorization mode directly instead of relying on the
+     * default location + override/fallback behavior defined in the {@link ClientBuilder documentation above}.
+     *
+     * Authorization Mode could take one of values - SSWS (by default), PrivateKey (for OAUTH2).
+     *
+     * @param authorizationMode mode of authorization for requests to the Okta API server.
+     * @return the ClientBuilder instance for method chaining.
+     */
+    ClientBuilder setAuthorizationMode(AuthorizationMode authorizationMode);
+
+    /**
+     * Allows specifying a list of scopes directly instead of relying on the
+     * default location + override/fallback behavior defined in the {@link ClientBuilder documentation above}.
+     *
+     * @param scopes list of scopes for which the client requests access.
+     * @return the ClientBuilder instance for method chaining.
+     */
+    ClientBuilder setScopes(List<String> scopes);
```

Comment:
nit: this could probably be a `Set` as they _should_ be unique (and order doesn't matter)

Reply:
done.